packages/react-instantsearch-hooks-dom/src/widgets/__tests__/RangeInput.test.tsx

```diff
@@ -0,0 +1,189 @@
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import {
+  createMultiSearchResponse,
+  createSearchClient,
+  createSingleSearchResponse,
+} from '../../../../../test/mock';
+import { InstantSearchHooksTestWrapper, wait } from '../../../../../test/utils';
+import { RangeInput } from '../RangeInput';
+
+function getSearchClient() {
```

Comment:
It doesn't make much sense to have `get*` call `create*`. We could call it something like `createSearchClientWithFacetsStats()`.